src/debugger/nodeDebugWrapper.ts

```diff
@@ -158,7 +158,7 @@ new EntryPointHandler(ProcessType.Debugger).runApp(appName, () => version,
 
             // We do not permit arbitrary args to be passed to our process
             args.args = [
-                args.platform,
+                args.platform || "all",

```

Comment:
I decided using "all", instead of "both" because in the future we could probably support more mobile platforms.